src/composables/useGatewayFeatureSupported.ts

```diff
@@ -1,13 +1,18 @@
+import { computed } from 'vue'
+import { useInfoStore } from '@/stores/info'
 import { useGatewayFeatureSupported as sharedComposable } from '@kong-ui-public/entities-shared'
 import { config } from 'config'
 
 type Arguments = Parameters<typeof sharedComposable>[0]
 
+const useInfo = useInfoStore()
+const kongVersion = computed(() => useInfo.kongVersion)
```

Comment:
why do we need to wrap `useInfo.kongVersion` in `computed` here?

Reply:
'cause the method `sharedComposable` can't get original value here.

Reply:
![Uploading image.png…]()

Reply:
![image](https://github.com/Kong/kong-manager/assets/22835296/6ed884fb-246b-45a4-b14f-434eef6a397b)